src/components/Concentrator/ConcentratorRewards.tsx

```diff
@@ -39,20 +40,18 @@ export const ConcentratorRewards: FC<ConcentratorRewardsProps> = ({
         <div>
           <h1 className="text-sm">Concentrator</h1>
           <h2 className="font-semibold">
-            <span className="bg-gradient-to-r from-orange to-pink bg-clip-text text-transparent">
+            <GradientText>
               <ConcentratorTargetAssetSymbol
                 concentratorTargetAsset={concentratorTargetAsset}
               />
-            </span>
+            </GradientText>
           </h2>
         </div>
         <div>
           <dl className="text-right">
             <dt className="text-sm">APY</dt>
             <dd className="font-semibold">
-              <span className="bg-gradient-to-r from-orange to-pink bg-clip-text text-transparent">
-                23.35%
-              </span>
+              <GradientText>23.35%</GradientText>
```

Comment:
I assume it's hardcoded here because the concentrator module isn't finished yet?

Reply:
Yessir